cmd/root.go

```diff
@@ -4,11 +4,12 @@ Copyright © 2022 NAME HERE <EMAIL ADDRESS>
 package cmd
 
 import (
-	configflagutil "k8s.io/test-infra/prow/flagutil/config"
 	"log"
 	"os"
 	"reflect"
 
+	configflagutil "k8s.io/test-infra/prow/flagutil/config"
+
```

Comment:
No need for the empty line here.

Reply:
good catch, will check what `go fmt` did on this.